portfolio/src/main/webapp/favoriteFoods.css

```diff
@@ -0,0 +1,60 @@
+.food-container {
+    margin-top: 10px;
+    height: 100%;
+    width: 100%;
+}
+
+.columns {
+    width:20%; 
+    height:100%;
+    position:absolute; 
+    -moz-transition:all 0.5s linear;
+    border: 0.5px solid black;
+    border-radius: 5px;
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    transition-duration: 2s; 
+}
+
+.columns:hover {
+    width:50%;   
+    overflow:visible; 
+    z-index:1;
+}
+
+.col1 {
+    background-image: url(/images/cachapa.jpg);
+}
+
+.col2 {
+    background-image: url(/images/mangu.jpg); left:20%;
```

Comment:
move left:20%; to the next line
ditto for the other col's

https://google.github.io/styleguide/htmlcssguide.html#Block_Content_Indentation